File: authority-portal-backend/authority-portal-api/src/main/java/de/sovity/authorityportal/api/model/CaasAvailabilityResponse.java
package de.sovity.authorityportal.api.model;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Schema(description = "Response for the availability of free CaaS")
public class CaasAvailabilityResponse {
    @Schema(description = "CaaS limit for your organization")
    private Integer limit;

    @Schema(description = "CaaS usage for your organization")
    private Integer current;
}

File: authority-portal-backend/authority-portal-api/src/main/java/de/sovity/authorityportal/api/model/CaasAvailabilityResponse.kt
package de.sovity.authorityportal.api.model

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Response for the availability of free CaaS")
data class CaasAvailabilityResponse(
    @field:Schema(description = "CaaS limit for your organization")
    val limit: Int,

    @field:Schema(description = "CaaS usage for your organization")
    val current: Int
)
